src/modules/townhalls/townhalls.ts

```diff
@@ -66,8 +72,130 @@ export async function getBillInfo(townhallId: string) {
     const townhall = await Collections.Townhalls().findOne({
         _id: new ObjectID(townhallId),
     });
+
     if (!townhall) throw new Error('Invalid Townhall ID');
 
-    // eventually after doing some other requests
-    return {};
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Leave more comments on distinct sections of code
Ex. // this extracts the vote information and puts them into the appropriate bucket -- senate or house vote